op-acceptance-tests/tests/base/leadership_transfer_test.go

```diff
@@ -0,0 +1,139 @@
+package base
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-conductor/consensus"
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-devstack/stack"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/stretchr/testify/require"
+)
+
+type conductorWithInfo struct {
+	stack.Conductor
+	info consensus.ServerInfo
+}
+
+// TestLeadershipTransfer checks if the leadership transfer works correctly on the conductors
+func TestLeadershipTransfer(gt *testing.T) {
```

Comment:
nit: Would `TestConductorLeadershipTransfer` be a more apt name here?

Reply:
Though leadership transfers imply conductor's only, I do like the more verbose naming, let me add it :)